doc/classes/FileDialog.xml

```diff
@@ -70,7 +70,7 @@
 			The dialog's open or save mode, which affects the selection behavior. See [enum FileMode].
 		</member>
 		<member name="filters" type="PackedStringArray" setter="set_filters" getter="get_filters" default="PackedStringArray()">
-			The available file type filters. For example, this shows only [code].png[/code] and [code].gd[/code] files: [code]set_filters(PackedStringArray(["*.png ; PNG Images","*.gd ; GDScript Files"]))[/code].
+			The available file type filters. For example, this shows only [code].png[/code] and [code].gd[/code] files: [code]set_filters(PackedStringArray(["*.png ; PNG Images","*.gd ; GDScript Files"]))[/code]. Multiple file types can also be specified in a single filter. [code]"*.png, *.jpg, *.jpeg ; Supported Images"[/code] will show both [code]png[/code] and [code]jpeg[/code] when selected.
```

Comment:
You list three extensions so "both `png` and `jpeg`" doesn't match the example. It should either list the three extensions again, or instead refer to the image format names: "both PNG and JPEG files".